packages/dds/tree/src/feature-libraries/defaultTransaction.ts

```diff
@@ -23,26 +23,27 @@ export function runSynchronousTransaction<TEditor extends ProgressiveEditBuilder
 	// This revision number is solely used within the scope of this transaction for the purpose of
```

Comment:
Consider updating the comment. Sorry GH isn't letting me make a suggestion here.
```
	// These revision numbers are solely used within the scope of this transaction for the purpose of
```

Reply:
There we go:
```suggestion
	// These revision numbers are solely used within the scope of this transaction for the purpose of
```